holidays/countries/malaysia.py

```diff
@@ -101,31 +107,29 @@ def __init__(
         See parameters and usage in :py:class:`HolidayBase`.
         """
         self.cnls = _ChineseLuniSolar()
+        ChineseCalendarHolidays.__init__(self)
```

Comment:
As you're migrating to `ChineseCalendarHolidays` you can get rid of `self.cnls` and use `self._chinese_calendar` instead.